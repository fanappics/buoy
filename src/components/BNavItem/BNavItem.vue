<template>
  <a class="b-nav-item">
    <slot></slot>
  </a>
</template>

<script>
export default {
  name: 'b-nav-item',
  props: {

  }
}
</script>

<style lang="stylus" scoped>
  a.b-nav-item
    padding 5px
    border-radius 2px
    text-decoration none
    color tomato
    &:hover
      background-color rgba(133, 133, 133, 0.2)
</style>
