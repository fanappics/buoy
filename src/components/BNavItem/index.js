import BNavItem from './BNavItem.vue'

/* istanbul ignore next */
BNavItem.install = function install (Vue) {
  Vue.component(BNavItem.name, BNavItem)
}

export default BNavItem
