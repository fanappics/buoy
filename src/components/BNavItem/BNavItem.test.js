import { shallow } from 'vue-test-utils'
import BNavItem from './BNavItem'

describe('BNavItem', () => {
  it('should render nav item', () => {
    const wrapper = shallow(BNavItem)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
