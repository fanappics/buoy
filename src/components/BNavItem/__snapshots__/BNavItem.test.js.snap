// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BNavItem should render nav item 1`] = `<a class="b-nav-item"></a>`;
